packages/teams-js/src/private/copilot.ts

```diff
@@ -0,0 +1,54 @@
+import { ensureInitialized } from '../internal/internalAPIs';
+import { errorNotSupportedOnPlatform } from '../public/constants';
+import { runtime } from '../public/runtime';
+import { AppEligibilityInformation } from './interfaces';
+
+/**
+ * @beta
+ * @hidden
+ * Namespace to delegate M365 chat app specific APIs
+ * @internal
+ * Limited to Microsoft-internal use
+ */
+export namespace copilot {
+  /**
+   * @beta
+   * @hidden
+   * User information required by specific apps
+   * @internal
+   * Limited to Microsoft-internal use
+   */
+  export namespace eligibility {
```

Comment:
>eligibility [](http://example.com/codeflow?start=19&length=11)

Out of curiosity, why are we immediately making this a subcapability?

Reply:
So, we can keep copilot related APIs into one capability. If we have any other APIs for copilot in future that is not related to `getEligibilityInfo` API, we would want a separate module for that. It would avoid the need to move the existing changes to a subcapability at that time (and will avoid the breaking changes).

Also with a subcapability,  the `isSupported` function will be called on `copilot.eligibility.isSupported` rather than `copilot.isSupported`. 

@erinha

Reply:
Based on how Copilot related features evolve, it seemed prudent to not put an API in the top-level capability as we currently don't have any functions that are appropriate for this from a naming perspective. By putting it in a sub-capability, we reserve the top-level capability for future use.